### PR TITLE
Add minimal states dropdown

### PR DESCRIPTION
**Pull request by alexsoble**
## Screenshot
Screen Shot 2021-12-14 at 10 04 43 AM

## What this does

+ Uses query params to send the user to the relevant state page on the existing site

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ github_info:
 # Enables Private Eye functionality.
 # See https://github.com/18F/private-eye for more information.
 # Private Eye configuration is in assets/js/main.js
-private_eye: true
+private_eye: false
```
Comment (alexsoble) [inline _config.yml:29]: This is a JS library we don't need; it's for links that go to private pages, like in the TTS Handbook.

File: _includes/components/hero.html [hunk 2]
```diff
@@ -11,15 +11,77 @@ <h1 class="usa-hero__heading">
         {% if hero.callout.alt %}
         <span class="usa-hero__heading--alt">{{ hero.callout.alt }}</span>
         {% endif %}
-        {{ hero.callout.text | default: hero.callout }}
```
Comment (alexsoble) [inline _includes/components/hero.html:14]: Got hacky with the `hero.html` component here ... we can clean this up later if we need to and name it something better and more specific like `states-dropdown-hero.html`.

File: _includes/components/hero.html [hunk 3]
```diff
@@ -11,15 +11,77 @@ <h1 class="usa-hero__heading">
...
+
+      <br/>
 
       {% if hero.button %}
-      <a class="usa-button"
-        href="{{ hero.button.href | relative_url }}">
+      <a
+        id="find-child-care-link"
+        class="usa-button disabled-link"
+        href="javascript:void(0);">
```
Comment (alexsoble) [inline _includes/components/hero.html:84]: Dynamically update this `href` based on the user select box

File: assets/js/states-dropdown.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      // user selected a state or territory; make link active and set correct url
+      findChildCareLink.classList.remove('disabled-link');
+      findChildCareLink.href = 'https://childcare.gov/state-resources?state=' + selected;
+    } else {
+      // user un-selected state or territory; disable the link and set href to void
+      findChildCareLink.classList.add('disabled-link');
+      findChildCareLink.href = 'javascript:void(0);';
+    }
+  });
+}
```
Comment (alexsoble) [inline assets/js/states-dropdown.js:18]: vanillaJS, want to avoid frameworks since this is pretty much the only interactivity needed on the site

**Comment by caleywoods** [comment]
Looks good to me.

**Pull request merged**